app/views/users/confirmations/show.html.haml

```diff
@@ -0,0 +1,29 @@
+.left-section
+  .left-container
+    .img-smilies
+      = image_tag 'smilies.png'
+    .text-container
+      %p
+        %strong
+          = "Hi, #{resource.name}"
+      %p Welcome to
+      %p 
+        %strong TARGET
+    = form_for(User.new, as: resource_name, url: session_path(resource_name)) do |f|
+      .field
+        = f.label :email, class: 'principal-label'
+        = f.email_field :email, autofocus: true, value: resource.email,  class: 'sing-up-field'
+      .field
+        = f.label :password, class: 'principal-label'
+        = f.password_field :password, autocomplete: "off", class: 'sing-up-field'
```

Comment:
Try to use the same symbol for strings, always single quotes or always double quotes. Choose one convention and keep it along the project.